tests/e2e/crypto.go

```diff
@@ -548,4 +560,76 @@ var _ = ginkgo.Describe("[csi-supervisor] [encryption] Block volume encryption",
 		ginkgo.By("12. Validate VM [6] is encrypted with first encryption key [1]")
 		validateVmToBeUpdatedWithEncryptedKey(ctx, vmopClient, vm.Namespace, vm.Name, keyProviderID, keyID1)
 	})
+
+	/*
+		Steps:
+			1. Generate first encryption key
+			2. Generate second encryption key
+			3. Create first EncryptionClass with encryption key [1]
+			4. Create second EncryptionClass with encryption key [2]
+			5. Creating PVC with first EncryptionClass [3]
+			6. Validate PVC volume [5] is encrypted with first encryption key [1]
+			7. Bring vsan-service down
+			8. Update PVC with second EncryptionClass [4]
+		    9. Bring vsan-service up
+			10. Validate PVC volume [5] is encrypted with second encryption key [2]
+	*/
+	ginkgo.It("Verify PVC encryption when vsan-health is down", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		ginkgo.By("1. Generate first encryption key")
+		keyID1 := e2eVSphere.generateEncryptionKey(ctx, keyProviderID)
+
+		ginkgo.By("2. Generate second encryption key")
+		keyID2 := e2eVSphere.generateEncryptionKey(ctx, keyProviderID)
+
+		ginkgo.By("3. Create first EncryptionClass with encryption key [1]")
+		encClass1 := createEncryptionClass(ctx, cryptoClient, namespace, keyProviderID, keyID1, false)
+		defer deleteEncryptionClass(ctx, cryptoClient, encClass1)
+
+		ginkgo.By("4. Create second EncryptionClass with encryption key [2]")
+		encClass2 := createEncryptionClass(ctx, cryptoClient, namespace, keyProviderID, keyID2, false)
+		defer deleteEncryptionClass(ctx, cryptoClient, encClass2)
+
+		ginkgo.By("5. Creating PVC with first EncryptionClass [3]")
```

Comment:
remove such step number from comments

Reply:
With these step numbers it is easy to track and understand if any failure